### DISCUSSION TITLE
Motor voltage for /wheel_status current_ma_left/right?  Optimal power efficiency Speed?

### DISCUSSION THREAD

**Discussion by slowrunner** [iRobotEducation/create3_docs#476] [Questions] [answered]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Computer(s) Model(s) and Operating System(s)

n/a

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

n/a

### Which RMW is your robot running?

I don't know

### Does your robot have an assigned namespace? If so, please share.

n/a

### Is the robot connected to a network? If so, what is the network type?

n/a

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

/wheel_status topic will report each motor's current in milliamps.   
What is the nominal voltage I should use to estimate the power consumption?  

My current non-Create3 robot can extend its playtime significantly by not traveling at max speed;  
Has anyone determined the stock Create3 optimal power efficiency speed?

**Comment by shamlian** [comment, marked as answer]
The robot uses the robot's battery voltage to run the motors. You can get the battery voltage from the `/battery_state` topic (it changes over time).
